src/gluonts/dataset/common.py

```diff
@@ -287,7 +288,7 @@ class Config:
         arbitrary_types_allowed = True
 
     freq: Union[str, pd.DateOffset]
-    name: str = "start"
+    name: str = FieldName.START
```

Comment:
Could we split these renaming-changes off into a separate PR, which we quickly merge?

Reply:
ok

Reply:
Thanks!